### PR TITLE
Vote for members and proposals

### PR DESCRIPTION
**Pull request by malnakli**

### CODE REVIEW

File: congress.sol [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        else 
+            p.voteCount--;
+        
+    }
+
+    function newProposal(string _des) validMember external returns (uint proposalId) {
+        return proposals.push(Proposal(_des,0, now + _debatingPeriodInMinutes)) - 1;
+    }
+     
+    //NOTE: I don't see the use of this function since proposals are public.
```
Comment (gorillya) [inline congress.sol]: how are you going to pull the proposals to the front end if you dont have a function to view proposals? not saying this is the right function.

File: member.sol [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        _members[member] = false;
+        numOfMembers--;
+        // when a member is removed all his proposal votes should be removed as well
+       
+        // uint i = 0;
+        // for (; i < proposals.length;i++) {
+        //     proposals[i].memberVoted[member] = false;
+        // }
+
+        RemveMember(member);
```
Comment (gorillya) [inline member.sol]: not RemoveMember ??

File: member.sol [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+        // since the new member is valid member, he or she can vote for himself/herself 
+        _memberVote[member].memberVoted[member] = true;
+        _memberVote[member].voteCount++;
+        AddMember(member);
+        
+    }
+
+    function _removeMember(address member) private {
+        /**
+         check that if all members have voted 'no' to a member exculed the member self.
```
Comment (gorillya) [inline member.sol]: "excluding"

File: congress.sol (hunk 1 above)
Comment (malnakli) [reply congress.sol]: Does `web3.js` have access to proposals array?

Comment (gorillya) [reply congress.sol]: I will figure that out today

Comment (eftakhairul) [reply congress.sol]: When it's public web3.js can access it.  Let say we have an instance of Congress: congressContract. Then we can  call ```await congressContract.proposals.call(0);```

**Comment by eftakhairul** [comment]
Seems okay. If required, we can modify again.Merged by Eftakhairul

**Pull request merged**